app/decorators/classification_decorator.rb

```diff
@@ -1,5 +1,9 @@
 class ClassificationDecorator < MiqDecorator
   def self.fonticon
-    'pficon pficon-folder-close'
+    super
```

Comment:
This is totally wrong :disappointed:, you applied [my comment](https://github.com/ManageIQ/manageiq-ui-classic/pull/160#discussion_r114299157) on the wrong line. 
```ruby
def self.fonticon
  'pficon pficon-folder-close'
end

def fonticon
  category? ? super : 'fa fa-tag'
end
```
The comment is usually related to the line RIGHT BEFORE the comment.